docs/usage.md

```diff
@@ -393,3 +393,33 @@ We recommend adding the following line to your environment to limit this (typica
 ```bash
 NXF_OPTS='-Xms1g -Xmx4g'
 ```
+
+### Running the pipeline without Internet access
+
+The pipeline and container images can be downloaded using [nf-core tools](https://nf-co.re/docs/usage/offline).
+Here is an example command to download pipeline version 1.1.0 with singularity images:
+
+```
+nf-core download \
```

Comment:
When I run that command with the latest version of `nf-core` (v2.9), I get:
```
 No such option: --container (Possible options: --container-cache-index, --container-library,     │
│ --container-system)
```

Reply:
Sorry about that. Seems they have changed the CLI since I downloaded nf-core tools

Reply:
I would just refer to the general docs on the website

Reply:
Ok, makes sense. I will remove the example.